src/client/pages/app.tsx

```diff
@@ -28,36 +28,31 @@ const EOP = Client({
   }),
 });
 
-class App extends React.Component {
-  static get propTypes() {
-    return {
-      match: PropTypes.object,
-    };
-  }
-
-  constructor(props) {
-    super(props);
-    const { params } = props.match;
-    this.state = {
-      game: params.game,
-      id: params.id,
-      secret: params.secret,
-    };
-  }
-
-  render() {
-    const playerId = this.state.id.toString();
-    return (
-      <div className="player-container">
-        <EOP
-          matchID={this.state.game}
-          credentials={this.state.secret}
-          playerID={playerId === SPECTATOR ? undefined : playerId}
-        />
-        <div className="cornucopiacard"></div>
-      </div>
-    );
-  }
+interface MatchParams {
+  game: string;
+  id: string;
+  secret: string;
 }
 
+type AppProps = RouteComponentProps<MatchParams>;
+
+const App: FC<AppProps> = ({ match }) => {
+  const [game] = useState(match.params.game);
```

Comment:
@ChristophNiehoff I'm not sure if there is any sense in having these 3 states. I just kept that from how it was before. But the state never seems to be changed, so I suppose we could also get rid of it all together?